arlo_server/rounds.py

```diff
@@ -121,7 +122,22 @@ def sample_ballots(election: Election, round: Round, sample_size: int):
 
 @app.route("/election/<election_id>/round", methods=["GET"])
 @with_election_access
-def list_rounds(election: Election):
+def list_rounds_audit_admin(election: Election):
+    rounds = sorted(election.rounds, key=lambda r: r.round_num)
```

Comment:
something we can do by default in the db relationship.

Reply:
yes, thanks!